gulpfile.js

```diff
@@ -11,15 +11,15 @@ const fileArgs = require('yargs').argv;
 // Directories to search SCSS files to compile. By default, node-sass does not
 // compile files that begin with _.
 const scssFilePaths = [
-  "web/modules/custom/**/*.scss",
-  "web/themes/custom/**/*.scss",
+  "docrootmodules/custom/**/*.scss",
```

Comment:
did you mean to have `docroot/modules/` and `docroot/themes/` instead?

Reply:
Oops! Thanks